### ISSUE TITLE
steam-glib.h missing from 1.4.0 tarball

### ISSUE THREAD

**Issue by silfreed** [jgeboski/bitlbee-steam#113]
It looks like the steam-glib.h file is missing from the 1.4.0 tarball:

```
Making all in steam
make[1]: Entering directory `/builddir/build/BUILD/bitlbee-steam-1.4.0/steam'
  CC       steam_la-steam.lo
  CC       steam_la-steam-api.lo
  CC       steam_la-steam-crypt.lo
  CC       steam_la-steam-http.lo
  CC       steam_la-steam-json.lo
  CC       steam_la-steam-user.lo
  CC       steam_la-steam-util.lo
In file included from steam-util.c:21:0:
steam-util.h:30:24: fatal error: steam-glib.h: No such file or directory
 #include "steam-glib.h"
```
```
$ tar tzf bitlbee-steam-1.4.0.tar.gz  | grep glib
$
```

**Comment by jgeboski** [comment]
Surprised it took this long from someone to report this, I guess a lot of people just use the Git tree. This has been resolved as of [1.4.1](https://github.com/jgeboski/bitlbee-steam/releases/tag/v1.4.1).

**Comment by dgw** [comment]
Hey, Git tree makes it super easy to check for updates. Nice and scriptable if I need to, too.
